### ISSUE TITLE
Changing slide width on the fly

### ISSUE THREAD

**Issue by nielsvantilborg** [kenwheeler/slick#2656]
Would love to have the functionality to have a single slide with a different width than the other slides. "Variable Width" doesn't do the trick because I want to change the width of the slide on the fly. The problem that occurs is that the slick slider doesn't except a slide to change width when you click an element. 

Example: http://frontend.develop.dailycms.com/partSlickSlider.html#partSlickSliderAlternativeAnchor

When you click another images you see it slide, but it looks buggy/it's flickering.

Looking forward for a solution. We need some kind of adaptive width instead of height :)

**Comment by pitops** [comment]
hey i have the same problem any solution?

**Comment by nielsvantilborg** [comment]
Not really. We modified/extended slick with new properties to make it happen.
